src/phoenix/server/api/types/Span.py

```diff
@@ -452,3 +451,32 @@ class _SpanIO:
     llm_input_messages: Any
     llm_output_messages: Any
     retrieval_documents: Any
+
+
+model_provider_to_model_prefix_map: dict[GenerativeProviderKey, list[str]] = {
```

Comment:
I wonder if this stuff should be the responsibility of the playground registry, or the playground clients, instead of the Span

Feels like it may be easier to discover and then update that way, particularly when adding new models support to the providers

Reply:
moved into the generative provider type file so that it is colocated